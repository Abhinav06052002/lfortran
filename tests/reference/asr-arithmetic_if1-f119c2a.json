{
    "basename": "asr-arithmetic_if1-f119c2a",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/arithmetic_if1.f90",
    "infile_hash": "41667893ffab9b073a6c0674e1821f3009b9c55fe383c44dd863ac77",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-arithmetic_if1-f119c2a.stderr",
    "stderr_hash": "25399056a213308b4af5f57124674c9a568bd10efa1f20ada7663b81",
    "returncode": 2
}