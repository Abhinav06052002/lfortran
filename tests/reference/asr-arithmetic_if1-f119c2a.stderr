semantic error: Arithmetic if (x) requires an integer or real for `x`
 --> tests/errors/arithmetic_if1.f90:5:1
  |
5 | if ("yy") 1, 2, 3
  | ^^^^^^^^^^^^^^^^^ 

semantic error: Arithmetic if (x) requires an integer or real for `x`
 --> tests/errors/arithmetic_if1.f90:5:5
  |
5 | if ("yy") 1, 2, 3
  |     ^^^^ 
